### PR TITLE
set services dynamically

### PR DESCRIPTION
**Pull request by blu3beri**
- feat(ios): set the services dynamically
- feat(android): explicitly supply the uuids

### CODE REVIEW

File: android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt [hunk 1]
```diff
@@ -10,30 +10,59 @@ import androidx.annotation.RequiresPermission
...
+        try {
+            this.centralManager = CentralManager(context)
+            promise.resolve(null)
+        } catch (e: Exception) {
+            promise.reject("error", e)
+        }
+    }
+
+    @ReactMethod
+    @RequiresPermission(value = "android.permission.BLUETOOTH_CONNECT")
```
Comment (morrieinmaas) [inline android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt:42]: is BLUETOOTH_CONNECT the only permissions required?

Comment (blu3beri) [reply android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt:42]: For this method, yes.

**Review by morrieinmaas — APPROVED** [review]
LGTM!

**Comment by rapaktech** [comment]
LGTM! Let get this merged and released!

**Comment by blu3beri** [comment]
@TimoGlastra could you review this?

**Comment by TimoGlastra** [comment]
I've created a new React Native and added @morrieinmaas as a maintainer. Should be good to merge now

**Pull request merged**